task/test-github

```diff
@@ -73,6 +73,8 @@ def mockServer():
                 issues_ = issues
                 if "state=open" in self.path:
                     issues_ = [i for i in issues_ if i["state"] == "open"]
+                if "since=" in self.path:
+                    issues_ = [i for i in issues_ if "created_at" not in i.keys() and "closed_at" not in i.keys()]
```

Comment:
That's a little weird, but as we don't track the modification time, good enough for now.

Reply:
it is super weird, but that is pretty much the same as it worked until now... But these tests could be greatly improved.